stack.yaml.lock

```diff
@@ -5,22 +5,29 @@
 
 packages:
 - completed:
+    hackage: hedgehog-1.0.5@sha256:85a2d8df595c91b815e9d2ef7734189d7028ce16212e83f35b2221807d87770b,4368
     pantry-tree:
       sha256: 9189de078d9bddcf9b2bf6e6a11a7dbcc2fec962b081e0a4b58301b14f6e25a7
       size: 2575
-    hackage: hedgehog-1.0.5@sha256:85a2d8df595c91b815e9d2ef7734189d7028ce16212e83f35b2221807d87770b,4368
   original:
     hackage: hedgehog-1.0.5@sha256:85a2d8df595c91b815e9d2ef7734189d7028ce16212e83f35b2221807d87770b,4368
 - completed:
+    hackage: non-empty-text-0.1.1@sha256:5c757d9b27c3ef61f1620613a5e105a870801ad1addef7ade62a2ca1798fea9f,1996
```

Comment:
Was stack,yaml.lock not committed prior when stack.yaml would have been updated? I'm confused why this is here with no corresponding stack.yaml change.

Reply:
It looks like it wasn't. Some people on the team were confused earlier in the year about changes to the lock file so I think that at the time it was added someone avoided changing the lock file and that wasn't caught in the PR.